pandas/tests/io/json/test_ujson.py

```diff
@@ -997,9 +997,11 @@ def test_dataframe_nested(self, orient):
         }
         assert ujson.decode(ujson.encode(nested, **kwargs)) == exp
 
-    def test_dataframe_numpy_labelled(self, orient):
+    def test_dataframe_numpy_labelled(self, orient, request):
         if orient in ("split", "values"):
-            pytest.skip("Incompatible with labelled=True")
```

Comment:
@willayd is xfail the right mark for this?  i.e. is it something that needs attention